### PR TITLE
tweak timeseries example and update parfive keyword

### PR DESCRIPTION
**Pull request by nabobalis**
Should hide the parfive progress correctly this time.

### CODE REVIEW

**Comment by nabobalis** [comment]
I will want to check RTD before a merge.

**Comment by nabobalis** [comment]
Looks better for me and parfive is hidden now.

File: sunpy/map/map_factory.py [hunk 1]
```diff
@@ -160,7 +160,7 @@ def _read_file(self, fname, **kwargs):
         try:
             pairs = read_file(os.fspath(fname), **kwargs)
         except Exception as e:
-            msg = f"Failed to read {fname}."
+            msg = f"Failed to read {fname}"
```
Comment (nabobalis) [inline sunpy/map/map_factory.py:163]: This annoys me when filepaths have an extra "." at the end and I copy it and then try to open it or less the file in the terminal.

Comment (dstansby) [reply sunpy/map/map_factory.py:163]: Ooh yes 👍👍 👍 👍 👍 👍 👍 👍 I noticed this earlier today

File: examples/time_series/timeseries_example.py [hunk 2]
```diff
@@ -1,7 +1,7 @@
 """
-====================================
-Using `~sunpy.timeseries.TimeSeries`
-====================================
+===================
+Using `.TimeSeries`
```
Comment (dstansby) [inline examples/time_series/timeseries_example.py:3]: Still looks a bit odd on the gallery page, get rid of the link in the title altogether?
Screenshot 2022-11-06 at 12 18 50

Comment (nabobalis) [reply examples/time_series/timeseries_example.py:3]: Yeah, it does look pretty bad.

**Review by Cadair — APPROVED** [review]
LGTM with David's suggested title tweak.

File: examples/time_series/timeseries_example.py [hunk 3]
```diff
@@ -1,7 +1,7 @@
...
+===================
```
Comment (nabobalis) [inline examples/time_series/timeseries_example.py:4]:
```suggestion
================
Using TimeSeries
================
```

**Comment by nabobalis** [comment]
Fixed

**Comment by nabobalis** [comment]
I will handle this manually.

**Pull request merged**